examples/SimpleRagas/SimpleRagas.ipynb

```diff
@@ -0,0 +1,597 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Using RAGAS metrics in Braintrust to measure and improve RAG performance\n",
+    "\n",
+    "RAGAS is a popular framework for evaluating Retrieval Augmented Generation (RAG) applications, and were recently included in Braintrust's `autoevals` package, with improvements. You can read more about our deep dive into RAGAS metrics [here](https://braintrust-git-ragas-braintrustdata.vercel.app/docs/cookbook/Ragas-Retrieval). For this cookbook example, we're going to build a simple RAG application and show how to compute RAGAS metrics.\n",
```

Comment:
@ankrgyl Are you going to merge your ragas deep dive cookbook in? It's interesting!